build/CopyFilesToStagingDir.ps1

```diff
@@ -116,8 +116,8 @@ PublishFile $FullBuildOutput\DynamicDependencyLifetimeManager.ProxyStub\DynamicD
 PublishFile $FullBuildOutput\DynamicDependencyLifetimeManager.ProxyStub\DynamicDependencyLifetimeManager.ProxyStub.pdb $NugetDir\runtimes\win10-$Platform\native
 #
 # Native (not managed, AppLocal / no MSIX)
-PublishFile $FullBuildOutput\WindowsAppRuntime_BootstrapDLL\Microsoft.WindowsAppRuntime.Bootstrap.dll $NugetDir\runtimes\lib\native\$Platform
-PublishFile $FullBuildOutput\WindowsAppRuntime_BootstrapDLL\Microsoft.WindowsAppRuntime.Bootstrap.pdb $NugetDir\runtimes\lib\native\$Platform
+PublishFile $FullBuildOutput\WindowsAppRuntime_BootstrapDLL\Microsoft.WindowsAppRuntime.Bootstrap.dll $NugetDir\runtimes\win10-$Platform\native
```

Comment:
>Bootstrap.dll [](http://example.com/codeflow?start=88&length=13)

where do we put bootstrap.net.dll?

Reply:
Transport package = `build\ProjectReunion-BuildFoundation.yml` line 172

```
TargetFolder: '$(Build.ArtifactStagingDirectory)\FullNuget\lib\net5.0-windows10.0.18362.0'
```

The Aggregator pipeline needs to copy that from the Transport to the Nuget. I'm uncertain if it does (the logic's a bit complex, some of the wildcarding may already be handling it). Easy enough to tell once this goes in, the Transport package will have the file and if the built NuGet doesn't I'll whip up a PR for that in the Aggregator.

I just spoke with @ionvasilian. The collective order of operations...

1. IXP pipeline changes complete (currently in progress)
2. IXP's .props in the IXP Transport package is adjusted to not trip over this PR
3. This PR's checked in
4. (If necessary) Aggregator PR copying bootstrap.net.dll to the NuGet